src/LabelsTrait.php

```diff
@@ -14,23 +14,51 @@
 
 trait LabelsTrait
 {
+    /**
+     * Returns domain labels
+     *
+     * @param string $input
+     *
+     * @return string[]
+     */
     private function getLabels(string $input): array
     {
         return explode('.', $input);
     }
 
+    /**
+     * Returns domains labesl in reverse
+     *
+     * @param string $input
+     *
+     * @return string[]
+     */
     private function getLabelsReverse(string $input): array
     {
         return array_reverse($this->getLabels($input));
     }
 
+    /**
+     * Tell whether the submit domain contains more than one label
```

Comment:
Remove "submit". The method knows nothing about submission, just that it's testing a domain.